x-pack/plugins/security_solution/public/common/containers/sourcerer/get_sourcerer_data_view.ts

```diff
@@ -24,6 +25,8 @@ export const getSourcererDataView = async (
   return {
     id: dataViewData.id ?? '',
     title: dataViewData.getIndexPattern(),
+    indexFields: dataViewData.fields,
+    fields: dataViewData.fields,
```

Comment:
We no longer rely on the sourcerer api's so we need to return the fields property as part of the "sourcerer data view". Eventually we will be able to remove this code entirely as we will not need to worry about rendering a pattern list in the sourcerer ui.